pkg/operation/botanist/resources.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2018 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package botanist
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/gardener/gardener/pkg/utils"
+)
+
+// DeployReferencedResources reads all referenced resources from the Garden cluster and writes them to the Seed cluster.
+func (b *Botanist) DeployReferencedResources(ctx context.Context) error {
```

Comment:
I guess this function lacks the deletion of those resources that were part of the `.spec.resources[]` in a previous shoot version and are now removed.
Probably it makes sense to leverage the Gardener-Resource-Manager deployed in every seed's `garden` namespace. It listens for `ManagedResource`s with `.spec.class=seed`. You could create such an object and let the GRM manage the referred resources in the seed. This way you don't have to remember the resources yourself to know what to clean up.

Reply:
Done